batchflow/tests/torch_layers_test.py

```diff
@@ -0,0 +1,69 @@
+""" Test torch layers """
+import pytest
+
+import torch
+import numpy as np
+
+from batchflow.models.torch.layers.pooling import GlobalMaxPool, GlobalAvgPool, \
+    AdaptiveMaxPool, AdaptiveAvgPool, MaxPool, AvgPool, Pool
+
+TEST_DATA = []
```

Comment:
Add some comment on the contents of each item: smth like `each item contains input array, correct result, layer and its parameters` should do
Although I would change the order: layer on the first place

Reply:
Make every added item of 4 (or 5 in case of loop) lines to improve readability